vms/registration/password_validate.py

```diff
@@ -0,0 +1,21 @@
+def validate_password(password):
+	upper_case=False #counter for uppercase characters
+	lower_case=False #counter for lowercase characters
+	number=False #counter for numbers
+	special_character=False #counter for special characters
+	special_list=['!','#','$','%','&','(',')','*','+',',','-','.','/',':',';','<','=','>','?','@','[',']','\\','^','_','`','{','}','|','~','"']
+	for x in password:
+		if x.isupper(): #checking for uppercase characters
+			upper_case=True
+		if x.islower(): #checking  for lowercase characters
+			lower_case=True
+		if x.isdigit(): #checking for nummbers
+			number=True
+		if x in special_list: #checking for special characters
+			special_character=True
+		if (upper_case==True and lower_case==True and special_character==True)
```

Comment:
Did you execute this code? I think there is a syntax error here. It should be 
```
if ((upper_case==True and lower_case==True and special_character==True) 
    or (upper_case==True and lower_case==True and number==True) 
    or (number==True and lower_case==True and special_character==True) 
    or (upper_case==True and number==True and special_character==True)):
```
when there are conditions in more than one line you need to include them in a bracket.
Please check this.

Reply:
@m-murad  I'd forgot to save that- updated it now.

Reply:
One more thing
Although I'm new to the community and don't know much about how work is done here but according to the recent conversations on slack I was asked not to approve a PRs if it doesn't have tests for the new code. Can you please write test(s) for this.